examples/utils.py

```diff
@@ -99,13 +101,17 @@ def get_distributed_model(model_path):
     # NOTE: bloom-560m is the default here because that's the default model used in our
     # text generation server hack script.
     model_name_override = os.getenv("MODEL_NAME", "bloom-560m")
```

Comment:
```suggestion
    model_name_override = os.getenv("MODEL_NAME", "bigscience/bloom-560m")
```
Just noticed that this is defaulting to the wrong name